distribution/install-profiles/src/main/resources/features.xml

```diff
@@ -20,6 +20,7 @@
         <feature>security-services-app</feature>
         <feature>catalog-app</feature>
         <feature>search-ui-app</feature>
+        <feature>catalog-ui-app</feature>
```

Comment:
This one has been undergoing testing for multiple months and will be turned on by default.